src/main/java/org/beanpod/switchboard/util/DateUtil.java

```diff
@@ -0,0 +1,17 @@
+package org.beanpod.switchboard.util;
+
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.util.Date;
+
+/**
+ * Utility class that returns the date of the current time in UTC time zone Useful for injecting
+ * test values for current time in tests.
```

Comment:
that comment says it's for tests but you use it in a service.

Reply:
I removed that part of the comment. I put it because this class only exists so i can do dependency injection in tests, but it's probably more misleading than anything else.